src/MeoAsstGui/Resources/Localizations/ja-jp.xaml

```diff
@@ -140,12 +145,10 @@
     <system:String x:Key="DrGrandetTip">純正源石使用確認画面で理性が1回復するのを待って、純正源石を使用します</system:String>
 
     <system:String x:Key="SocialPtShop">FP交換</system:String>
-    <system:String x:Key="HighPriority">優先購入（OCR/セミコロンで区切）</system:String>
-    <!--日本語名は未対応？-->
-    <system:String x:Key="Blacklist">ブラックリスト（OCR/セミコロンで区切）</system:String>
-    <system:String x:Key="HighPriorityDefault">求人票；龍門幣</system:String>
-    <system:String x:Key="BlacklistDefault">炭素材；家具</system:String>
-    <!--日本語名は未対応？-->
+    <system:String x:Key="HighPriority">優先購入（例:龍門弊/セミコロンで区切）</system:String>
+    <system:String x:Key="Blacklist">ブラックリスト（例:純金/セミコロンで区切）</system:String>
```

Comment:
这个 `/` 号会不会引起误解，要不要改成
`優先購入（セミコロンで区切，例：龍門幣；求人票）`
“弊”也写错了，应该是“幣”，记得改一下哦

Will this `/` sign cause misunderstanding? Shall we change it to
`優先購入（セミコロンで区切，例：龍門幣；求人票）` ?
"弊" is also a wrong character, it should be "幣", don't forget to change it

Reply:
而且后面已经有 Default 值了，这个“例”可以不用加的。

Since there is already a Default value, this "例" can be removed.

Reply:
"弊"は純粋にミスですね・・・ご指摘有難うございます。
確かに区切りがあることで分かりづらいのも事実だと思いますので、その案で良いと思います。
ホワイトリストとブラックリストに関してはマニュアルには記載がなかったので例を追加しました。
無駄な質問を避けるためという狙いもありましたが、確かにデフォルトの記載で分かると言われればその通りです。

省きましょう。

/

The "弊" is purely a mistake... Thank you for pointing that out.
It is true that the delimitation makes it difficult to understand, so I think that idea is fine.
As for the white list and black list, I did not see anything in the manual, so I added an example.
I added the examples to avoid unnecessary questions, but it is true that the default description is enough to understand.

Let's omit it.